// *****************************************************************************
//
// This file is part of a MASA library or program.
// Refer to the included end-user license agreement for restrictions.
//
// Copyright (c) 2010 MASA Group
//
// *****************************************************************************

package MsgsAarToClient;

message MsgAarInformation {
    required string information = 1;
}

message MsgPlotResult {
    required uint32 identifier = 1;
    repeated float  values     = 2;
    required string error      = 3;
}

message MsgIndicator {
    required string name  = 1;
    required float  value = 2;
}

message MsgAarToClient {
    message Content {
        optional MsgAarInformation aar_information = 1;
        optional MsgPlotResult     plot_result     = 2;
        optional MsgIndicator      indicator       = 3;
    }
    optional int32   context = 1;
    optional Content message = 2;
}
